### PR TITLE
Add recipe for cuda-api-wrappers

### PR DESCRIPTION
**Pull request by awvwgk**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by awvwgk** [comment]
@conda-forge/help-c-cpp This header-only library recipe is ready for review.

File: recipes/cuda-api-wrappers/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/eyalroz/cuda-api-wrappers/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: fa30c9fe43a62f5a3fd82a5deb477838fbf0bf455c73a2d2bb5ab6284184900b
+
+build:
+  number: 0
+  skip: True  # [cuda_compiler != 'None']
```
Comment (xhochy) [inline recipes/cuda-api-wrappers/meta.yaml:14]: The failures are actually correct ;)
```suggestion
  skip: True  # [cuda_compiler == 'None']
```

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by xhochy** [comment]
Can you please fix the lint? Then it is good to go.

File: recipes/cuda-api-wrappers/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  commands:
+    - test -f ${PREFIX}/include/cuda/api.hpp
+    - test -f ${PREFIX}/include/cuda/rtc.hpp
+    - test -f ${PREFIX}/include/cuda/nvtx.hpp
+    - test -f ${PREFIX}/include/cuda/runtime_api.hpp
+    - test -f ${PREFIX}/lib/cmake/cuda-api-wrappers/cuda-api-wrappers-config.cmake
+    - test -f ${PREFIX}/lib/cmake/cuda-api-wrappers/cuda-api-wrappers-targets.cmake
+
+about:
+  host: https://github.com/eyalroz/cuda-api-wrappers
```
Comment (awvwgk) [inline recipes/cuda-api-wrappers/meta.yaml:35]:
```suggestion
  home: https://github.com/eyalroz/cuda-api-wrappers
```

**Review by xhochy — APPROVED** [review]

**Pull request merged**